x-pack/plugins/security_solution/public/detection_engine/rule_management/api/api.ts

```diff
@@ -299,6 +300,7 @@ export interface BulkActionResult {
   created: RuleResponse[];
   deleted: RuleResponse[];
   skipped: RuleResponse[];
+  backfilled: RuleResponse[];
```

Comment:
Looks like you extended this result type here with `backfilled` but ended up using `BulkScheduleBackfillActionResponse` directly, and `backfilled` is unused here.

Reply:
done